### ISSUE TITLE
💬 [Support Request] TBC database can't be updated using the InstallFullDB.sh script

### ISSUE THREAD

**Issue by LEO33SoFo** [cmangos/issues#3619]
### Core Version

mangos-tbc

### Support Request Details

Duplicate issue of https://discord.com/channels/362206349339262976/1180696273184112731.

I get 
`ERROR 1064 (42000) at line 1041: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'a WHERE guid NOT IN(SELECT guid FROM gameobject b WHERE a.guid=b.guid) AND guid ' at line 1`
when updating via **InstallFullDB.sh** at the line
`Applying Updates/0275_tbc_starter_zone_vein_overhaul.sql ... FAILED!`,
using _MySQL_ v5.7.41.

I have to manually import the pre-made SQL world dump from https://github.com/cmangos/tbc-db/releases/tag/latest to remedy this problem.

### Steps to Reproduce

1. Use _MySQL_ v5.7.41 as a database server
2. Run the file **InstallFullDB.sh** as per official instructions.
3. Check for the aforementioned issue.

### Crash Log

```
bash: !: event not found
  - Unziping TBCDB_1.10.0_ReturnOfTheVengeance.sql.gz ... SUCCESS
  - Extracting db information ... SUCCESS
[H[2J[3JChecking minimum requirement please wait...
[H[2J[3JWelcome to the CMaNGOS TBC databases manager
============================================

Source version            : [1.10.0] Return of the Vengeance
Last source content update: [0291_npc_vendor_limited_stock]
Database content version  : []

Warning: These DBs need core update: 'tbcmangos'          You can use option 3 to apply latest core update


> 1) Manage settings
> 2) Install fresh TBC-DB only to 'tbcmangos'
> 3) Install core updates only
> 4) Full installation (create all DB and MySQL user, root required)
> 5) Advanced DB management (root required)
> 6) Manage realm list
> 7) Manage your DB backup
> 9) Quit

[?2004hPlease enter your choice.....: 2
[?2004l
[H[2J[3J[?2004hAll previous changes will be lost. Type 'Install' if you are sure : Install
[?2004l
> Processing main TBC-DB release file "Return of the Vengeance" ... 
  - Applying TBCDB_1.10.0_ReturnOfTheVengeance.sql ... SUCCESS


> Trying to process TBC-DB updates
  - Applying Updates/0001_CDB-4352_LH-20171031151600_world.sql ... SUCCESS
  - Applying Updates/0002_creature_spawn_data_template_9989_name.sql ... SUCCESS
  - Applying Updates/0003_CDB-4353_dustwallow_mithril.sql ... SUCCESS
  - Applying Updates/0004_ccsdb_tbc_resistances.sql ... SUCCESS
  - Applying Updates/0005_mosshide_gnolls.sql ... SUCCESS
  - Applying Updates/0006_Core_s2467_01_mangos_creature_static_flags.sql ... SUCCESS
  - Applying Updates/0007_Well_Watcher_Solanian_breadcrumb_quests.sql ... SUCCESS
  - Applying Updates/0008_no_melee_flee_flag.sql ... SUCCESS
  - Applying Updates/0009_creature_template_movement.rooted.1.sql ... SUCCESS
  - Applying Updates/0010_sw_catherine_vendor.sql ... SUCCESS
  - Applying Updates/0011_WDB-4797_c_3520.sql ... SUCCESS
  - Applying Updates/0012_Rescue_From_Jaedenar_escort.sql ... SUCCESS
  - Applying Updates/0013_CDB-4355_blasted_lands_black_slayer.sql ... SUCCESS
  - Applying Updates/0014_CDB-4354_blasted_lands_felguard_sentry.sql ... SUCCESS
  - Applying Updates/0015_CDB-4356_blasted_lands_serpents_coil.sql ... SUCCESS
  - Applying Updates/0016_CDB-4357_data1_2280,2281,2282,2283,2284,5278_chest_loot.sql ... SUCCESS
  - Applying Updates/0017_CDB-4358_LH_20171031140930_world.sql ... SUCCESS
  - Applying Updates/0018_CDB-4359_blasted_lands_sungrass.sql ... SUCCESS
  - Applying Updates/0019_CDB-4360_LH-20180627203943_world.sql ... SUCCESS
  - Applying Updates/0020_CDB-4362_s.3417_thrash.sql ... SUCCESS
  - Applying Updates/0021_CDB_4363_blasted_lands_dreadmaul_post.pkt.sql ... SUCCESS
  - Applying Updates/0022_CDB-4364_blasted_lands_wildlife_1.sql ... SUCCESS
  - Applying Updates/0023_Core_s2468_01_mangos_visibility.sql ... SUCCESS
  - Applying Updates/0024_CDB-4365_blasted_lands_spiteflayer.sql ... SUCCESS
  - Applying Updates/0025_CDB_4366_blasted_lands_wildlife_2.sql ... SUCCESS
  - Applying Updates/0026_CDB-4367_blasted_solid_chest.sql ... SUCCESS
  - Applying Updates/0027_core_trainer_ability.sql ... SUCCESS
  - Applying Updates/0028_core_spawn_group_worldstate_expr.sql ... SUCCESS
  - Applying Updates/0029_CDB-4368_blasted_lands_mithril_deposit_1.sql ... SUCCESS
  - Applying Updates/0030_CDB-4369_blasted_lands_firebloom.sql ... SUCCESS
  - Applying Updates/0031_CDB-4370_blasted_lands_dreadscorn.sql ... SUCCESS
  - Applying Updates/0032_CDB-4371_blasted_lands_altar_of_storms.sql ... SUCCESS
  - Applying Updates/0033_CDB-4372_hinterlands_sharks.sql ... SUCCESS
  - Applying Updates/0034_CDB-4373_hinterlands_stranglekelp.sql ... SUCCESS
  - Applying Updates/0035_CDB-4374_blasted_lands_gromsblood.sql ... SUCCESS
  - Applying Updates/0036_DireMaulCrystalTotem_TotemAI.sql ... SUCCESS
  - Applying Updates/0037_ohf_lieutenant_drake.sql ... SUCCESS
  - Applying Updates/0038_LH-20180703023407_world.sql ... SUCCESS
  - Applying Updates/0039_ohf_captain_skarloc.sql ... SUCCESS
  - Applying Updates/0040_core_sync.sql ... SUCCESS
  - Applying Updates/0041_ohf_epoch_hunter.sql ... SUCCESS
  - Applying Updates/0042_karazhan_ai_rework.sql ... SUCCESS
  - Applying Updates/0043_LH-20180710193231_world.sql ... SUCCESS
  - Applying Updates/0044_karazhan_spectral_stable_hand.sql ... SUCCESS
  - Applying Updates/0045_LH-20180711115217_world.sql ... SUCCESS
  - Applying Updates/0046_LH-20190203022442_world.sql ... SUCCESS
  - Applying Updates/0047_VDB-20221217024824_world.sql ... SUCCESS
  - Applying Updates/0048_moonbrook_statics.sql ... SUCCESS
  - Applying Updates/0049_rfc_callforhelp.sql ... SUCCESS
  - Applying Updates/0050_WDB-4798_c.68.sql ... SUCCESS
  - Applying Updates/0051_WDB-4800_c.682_736.sql ... SUCCESS
  - Applying Updates/0052_VDB-20230108115505_world.sql ... SUCCESS
  - Applying Updates/0053_VDB-20230525175252_world.sql ... SUCCESS
  - Applying Updates/0054_VDB-20230314050619_world.sql ... SUCCESS
  - Applying Updates/0055_VDB-20220820203350_world.sql ... SUCCESS
  - Applying Updates/0056_bm_aeonus.sql ... SUCCESS
  - Applying Updates/0057_bm_chrono_lord_deja.sql ... SUCCESS
  - Applying Updates/0058_bm_temporus.sql ... SUCCESS
  - Applying Updates/0059_ac_shirrak.sql ... SUCCESS
  - Applying Updates/0060_ac_maladaar.sql ... SUCCESS
  - Applying Updates/0061_mt_pandemonius.sql ... SUCCESS
  - Applying Updates/0062_CDB-4375_silithus_golden_sansam.sql ... SUCCESS
  - Applying Updates/0063_CDB-4376_silithus_sungrass.sql ... SUCCESS
  - Applying Updates/0064_CDB-4377_eastern_plaguelands_mithril.sql ... SUCCESS
  - Applying Updates/0065_CDB-4378_eastern_plaguelands_small_thorium.sql ... SUCCESS
  - Applying Updates/0066_CDB-4379_eastern_plaguelands_rich_thorium.sql ... SUCCESS
  - Applying Updates/0067_mt_yor.sql ... SUCCESS
  - Applying Updates/0068_mt_tavarok.sql ... SUCCESS
  - Applying Updates/0069_sh_darkweaver_syth.sql ... SUCCESS
  - Applying Updates/0070_CDB-4380_blasted_lands_wildlife_3.sql ... SUCCESS
  - Applying Updates/0071_CDB-4381_blasted_lands_clack_the_reaver.sql ... SUCCESS
  - Applying Updates/0072_2.3.0_drums_world_loot.sql ... SUCCESS
  - Applying Updates/0073_CDB-4382_blated_lands_small_thorium_vein.sql ... SUCCESS
  - Applying Updates/0074_CDB-4383_swamp_of_sorrows_iron_deposit.sql ... SUCCESS
  - Applying Updates/0075_WDB-4807_q.231.sql ... SUCCESS
  - Applying Updates/0076_WDB-4809_q.75.sql ... SUCCESS
  - Applying Updates/0077_WDB-4811_MizjahRuins_StranglethornVale.sql ... SUCCESS
  - Applying Updates/0078_sh_anzu.sql ... SUCCESS
  - Applying Updates/0079_escort_c.8284.sql ... SUCCESS
  - Applying Updates/0080_CDB-4385_burning_steppes_mithril_deposit.sql ... SUCCESS
  - Applying Updates/0081_c.5914_faction.sql ... SUCCESS
  - Applying Updates/0082_sl_ambassador_hellmaw.sql ... SUCCESS
  - Applying Updates/0083_VDB-20220823192212_world.sql ... SUCCESS
  - Applying Updates/0083_sl_blackheart_the_inciter.sql ... SUCCESS
  - Applying Updates/0084_sl_grandmaster_vorpil.sql ... SUCCESS
  - Applying Updates/0085_sl_murmur.sql ... SUCCESS
  - Applying Updates/0086_CDB-4388_burning_steppes_small_thorium_vein.sql ... SUCCESS
  - Applying Updates/0086_sp_mennu_the_betrayer.sql ... SUCCESS
  - Applying Updates/0087_CDB-4405_dberrors_XarTi_7953_v2.sql ... SUCCESS
  - Applying Updates/0088_CDB-4412_burning_steppes_rich_thorium_vein.sql ... SUCCESS
  - Applying Updates/0089_CDB-4416_felwood_plaguebloom.sql ... SUCCESS
  - Applying Updates/0090_fel_creep_11514.sql ... SUCCESS
  - Applying Updates/0091_CDB-4417_silithus_moutain_silversage.sql ... SUCCESS
  - Applying Updates/0092_CDB-4418_ungoro_small_thorium_vein.sql ... SUCCESS
  - Applying Updates/0093_WDB-4820_sealed_scroll_case_i.33926.sql ... SUCCESS
  - Applying Updates/0094_WDB-4812_BarrierHills_TerokkarForest.sql ... SUCCESS
  - Applying Updates/0095_WDB-4821_message_in_a_bottle_i.6307.sql ... SUCCESS
  - Applying Updates/0096_WDB-4823_q.8731_8507.sql ... SUCCESS
  - Applying Updates/0097_c.15449,15620_stats.sql ... SUCCESS
  - Applying Updates/0098_claw_command_targeting.sql ... SUCCESS
  - Applying Updates/0099_underbog_bct.sql ... SUCCESS
  - Applying Updates/0100_mt_bct.sql ... SUCCESS
  - Applying Updates/0101_Eskhandar_c.14306.sql ... SUCCESS
  - Applying Updates/0101_ikis_bct.sql ... SUCCESS
  - Applying Updates/0102_CDB-4419_ashenvale_gromsblood.sql ... SUCCESS
  - Applying Updates/0102_rokmar.sql ... SUCCESS
  - Applying Updates/0103_CDB-4420_swamp_of_sorrows_mithril_deposit.sql ... SUCCESS
  - Applying Updates/0104_annora_malacrass_conditions.sql ... SUCCESS
  - Applying Updates/0105_CDB-4421_stranglethorn_vale_tin_vein.sql ... SUCCESS
  - Applying Updates/0105_quagmirran.sql ... SUCCESS
  - Applying Updates/0106_CDB-4422_westfall_tin_vein.sql ... SUCCESS
  - Applying Updates/0107_CDB-4423_q.5156_areatrigger.sql ... SUCCESS
  - Applying Updates/0108_Midsummer_bonfires.sql ... SUCCESS
  - Applying Updates/0109_WDB-4827_q.8924.sql ... SUCCESS
  - Applying Updates/0110_sv_thespia.sql ... SUCCESS
  - Applying Updates/0111_c.9451_loot.sql ... SUCCESS
  - Applying Updates/0112_CDB-4435_stranglethorn_vale_mithril_deposit.sql ... SUCCESS
  - Applying Updates/0113_CDB-4436_bookie_herod.sql ... SUCCESS
  - Applying Updates/0114_sv_mekgineer.sql ... SUCCESS
  - Applying Updates/0115_WDB-4834_q.9740.sql ... SUCCESS
  - Applying Updates/0116_WDB-4835_c.17528.sql ... SUCCESS
  - Applying Updates/0117_generic_scripts.sql ... SUCCESS
  - Applying Updates/0118_c.9451_ai.sql ... SUCCESS
  - Applying Updates/0119_o.175124_default_data.sql ... SUCCESS
  - Applying Updates/0120_sv_kalithresh.sql ... SUCCESS
  - Applying Updates/0121_VDB-20230527024300_world.sql ... SUCCESS
  - Applying Updates/0122_VDB-20221031075849_world.sql ... SUCCESS
  - Applying Updates/0123_VDB-20230217150752_world.sql ... SUCCESS
  - Applying Updates/0124_VDB-20190719174925_world.sql ... SUCCESS
  - Applying Updates/0125_VDB-20200501101359_world.sql ... SUCCESS
  - Applying Updates/0126_karazhan_callforhelp.sql ... SUCCESS
  - Applying Updates/0127_CDB-4437_steelsnap.sql ... SUCCESS
  - Applying Updates/0128_dberrors.sql ... SUCCESS
  - Applying Updates/0129_shh_nethekurse.sql ... SUCCESS
  - Applying Updates/0130_WDB-4845_random_stuff.sql ... SUCCESS
  - Applying Updates/0131_VDB-20200424151410_world.sql ... SUCCESS
  - Applying Updates/0132_VDB-20200424152628_world.sql ... SUCCESS
  - Applying Updates/0133_VDB-20200426205604_world.sql ... SUCCESS
  - Applying Updates/0134_VDB-20200531090345_world.sql ... SUCCESS
  - Applying Updates/0135_VDB-20210415162937_world.sql ... SUCCESS
  - Applying Updates/0136_WDB-4846_Defias_aggro_texts.sql ... SUCCESS
  - Applying Updates/0137_shh_kargath.sql ... SUCCESS
  - Applying Updates/0138_WDB-4796_blacksmithing.sql ... SUCCESS
  - Applying Updates/0139_WDB-4844_argent_dawn_quests.sql ... SUCCESS
  - Applying Updates/0140_WDB-4851_c.14496.sql ... SUCCESS
  - Applying Updates/0141_WDB-4858_c.550_aggro_texts.sql ... SUCCESS
  - Applying Updates/0142_WDB-4859_c.727.sql ... SUCCESS
  - Applying Updates/0143_WDB-4860_c.1337.sql ... SUCCESS
  - Applying Updates/0144_WDB-4861_c.2835.sql ... SUCCESS
  - Applying Updates/0145_WDB-4862_c.345.sql ... SUCCESS
  - Applying Updates/0146_WDB-4863_c.8416.sql ... SUCCESS
  - Applying Updates/0147_CDB-2669_gameobject_template_updates.sql ... SUCCESS
  - Applying Updates/0147_omrogg.sql ... SUCCESS
  - Applying Updates/0148_gruul_sounds.sql ... SUCCESS
  - Applying Updates/0149_arcatraz_bct.sql ... SUCCESS
  - Applying Updates/0150_VDB-20230625171757_world.sql ... SUCCESS
  - Applying Updates/0151_c.9938_creature_spell_list.sql ... SUCCESS
  - Applying Updates/0152_daliah.sql ... SUCCESS
  - Applying Updates/0153_CDB-1752_game_event_gameobject.sql ... SUCCESS
  - Applying Updates/0153_zereketh.sql ... SUCCESS
  - Applying Updates/0154_WDB-147_creature.sql ... SUCCESS
  - Applying Updates/0155_map_530_commoners.sql ... SUCCESS
  - Applying Updates/0156_steamvault_surger_patrols.sql ... SUCCESS
  - Applying Updates/0157_karazhan_cleanup.sql ... SUCCESS
  - Applying Updates/0158_shattered_halls_ai.sql ... SUCCESS
  - Applying Updates/0159_WDB-4872_TDB-0742_restore_nohover.sql ... SUCCESS
  - Applying Updates/0160_WDB-4865_q.333.sql ... SUCCESS
  - Applying Updates/0161_WDB-4866_Stormwind_updates.sql ... SUCCESS
  - Applying Updates/0162_WDB-4867_Zzarc_Vul_c.300.sql ... SUCCESS
  - Applying Updates/0163_broken_ones_classic.sql ... SUCCESS
  - Applying Updates/0164_CDB-4461_zulian_panther.sql ... SUCCESS
  - Applying Updates/0165_CDB-4462_LH-20170326174853_world.sql ... SUCCESS
  - Applying Updates/0166_CDB-4463_generic_run_on_relayscript.sql ... SUCCESS
  - Applying Updates/0167_guards_ais.sql ... SUCCESS
  - Applying Updates/0168_CDB-4464_zulgurub_speed.sql ... SUCCESS
  - Applying Updates/0169_CDB-4465_auras_7939_22766.sql ... SUCCESS
  - Applying Updates/0170_soccothrates.sql ... SUCCESS
  - Applying Updates/0171_skyriss.sql ... SUCCESS
  - Applying Updates/0172_botanica_bct.sql ... SUCCESS
  - Applying Updates/0173_c.68_random_movement.sql ... SUCCESS
  - Applying Updates/0174_WDB-4883_c.20290.sql ... SUCCESS
  - Applying Updates/0175_high_botanist_freywinn.sql ... SUCCESS
  - Applying Updates/0176_TrinityCore-2020_01_09_01_world.sql ... SUCCESS
  - Applying Updates/0176_laj.sql ... SUCCESS
  - Applying Updates/0177_kilsorrow_moneyloot.sql ... SUCCESS
  - Applying Updates/0178_TrinityCore-2022_10_30_01_world.sql ... SUCCESS
  - Applying Updates/0179_TrinityCore-2022_10_30_07_world.sql ... SUCCESS
  - Applying Updates/0180_TrinityCore-2022_11_05_01_world.sql ... SUCCESS
  - Applying Updates/0181_TrinityCore-2022_06_20_00_world.sql ... SUCCESS
  - Applying Updates/0182_TrinityCore-2021_10_28_00_world.sql ... SUCCESS
  - Applying Updates/0183_AC-rev_1636103884460644100.sql ... SUCCESS
  - Applying Updates/0183_ekkorash_spell_list.sql ... SUCCESS
  - Applying Updates/0184_arkelos.sql ... SUCCESS
  - Applying Updates/0185_arkelos_the_guardian.sql ... SUCCESS
  - Applying Updates/0186_truth_unorbed.sql ... SUCCESS
  - Applying Updates/0187_WDB-4884_c.4973.sql ... SUCCESS
  - Applying Updates/0188_WDB-4885_q.1284.sql ... SUCCESS
  - Applying Updates/0189_WDB-4890_c.1057_CSL.sql ... SUCCESS
  - Applying Updates/0190_quest_triangulation_point.sql ... SUCCESS
  - Applying Updates/0191_culuthas.sql ... SUCCESS
  - Applying Updates/0192_respawn_the_bone_wastes_wildlife.sql ... SUCCESS
  - Applying Updates/0193_respawn_go.182584.sql ... SUCCESS
  - Applying Updates/0194_respawn_c.22095,22307.sql ... SUCCESS
  - Applying Updates/0195_WDB-4900_q.10169.sql ... SUCCESS
  - Applying Updates/0196_WDB-4901_c.1847.sql ... SUCCESS
  - Applying Updates/0197_WDB-4902_q.2969.sql ... SUCCESS
  - Applying Updates/0198_c.19849,19851.sql ... SUCCESS
  - Applying Updates/0199_respawn_go.187073.sql ... SUCCESS
  - Applying Updates/0200_thorngrin.sql ... SUCCESS
  - Applying Updates/0201_warp_splinter.sql ... SUCCESS
  - Applying Updates/0202_mechanar_bct.sql ... SUCCESS
  - Applying Updates/0203_mechano_lord_capacitus.sql ... SUCCESS
  - Applying Updates/0204_sepethrea.sql ... SUCCESS
  - Applying Updates/0205_pathaleon.sql ... SUCCESS
  - Applying Updates/0206_affect_modernization.sql ... SUCCESS
  - Applying Updates/0207_bf_bct.sql ... SUCCESS
  - Applying Updates/0207_doom_lord_kazzak.sql ... SUCCESS
  - Applying Updates/0207_shh_callforhelp_radius.sql ... SUCCESS
  - Applying Updates/0208_shh_nethekurse_spell_target.sql ... SUCCESS
  - Applying Updates/0209_full_triangle.sql ... SUCCESS
  - Applying Updates/0210_quest_warn_bolvar.sql ... SUCCESS
  - Applying Updates/0211_CDB-4490_VDB-20230210082850_world.sql ... SUCCESS
  - Applying Updates/0211_doomwalker.sql ... SUCCESS
  - Applying Updates/0211_shh_gladiators.sql ... SUCCESS
  - Applying Updates/0212_CDB-4491_VDB-20220613183352_world.sql ... SUCCESS
  - Applying Updates/0213_brewfest.sql ... SUCCESS
  - Applying Updates/0214_demolitionist_legoso.sql ... SUCCESS
  - Applying Updates/0215_collidus_the_watcher.sql ... SUCCESS
  - Applying Updates/0216_kara_moroes_adds.sql ... SUCCESS
  - Applying Updates/0217_hfr_bct.sql ... SUCCESS
  - Applying Updates/0218_WDB-4918_c.16854.sql ... SUCCESS
  - Applying Updates/0219_hydross.sql ... SUCCESS
  - Applying Updates/0220_thrall_ohf.sql ... SUCCESS
  - Applying Updates/0221_the_eye_bct.sql ... SUCCESS
  - Applying Updates/0222_spectral_charger.sql ... SUCCESS
  - Applying Updates/0223_ambassador_jerrikar.sql ... SUCCESS
  - Applying Updates/0224_karazhan_ai_3.sql ... SUCCESS
  - Applying Updates/0225_ramparts_ai.sql ... SUCCESS
  - Applying Updates/0226_CDB-4492_spell_facing.sql ... SUCCESS
  - Applying Updates/0227_CDB-4492_VDB-20220629204830_world.sql ... SUCCESS
  - Applying Updates/0228_CDB-4493_VDB-20191006132107_world.sql ... SUCCESS
  - Applying Updates/0229_anthion_harmon_gossip.sql ... SUCCESS
  - Applying Updates/0230_the_newest_update_brd_suckit.sql ... SUCCESS
  - Applying Updates/0231_spell_list_target_fix.sql ... SUCCESS
  - Applying Updates/0232_s.7131.sql ... SUCCESS
  - Applying Updates/0233_WDB-5124_c.727_13076.sql ... SUCCESS
  - Applying Updates/0234_WDB-5127_c.8404.sql ... SUCCESS
  - Applying Updates/0235_WDB-4735_Orgrimmar_waypoints.sql ... SUCCESS
  - Applying Updates/0236_WDB-5131_c.14270.sql ... SUCCESS
  - Applying Updates/0237_brd_05_aggro_bct.sql ... SUCCESS
  - Applying Updates/0238_the_challenge.sql ... SUCCESS
  - Applying Updates/0239_bodley.sql ... SUCCESS
  - Applying Updates/0240_geologist_larksbane.sql ... SUCCESS
  - Applying Updates/0241_soul_ashes_of_the_banished.sql ... SUCCESS
  - Applying Updates/0242_jeerings_spectres_essence.sql ... SUCCESS
  - Applying Updates/0243_ogre_warbeads.sql ... SUCCESS
  - Applying Updates/0244_bloodkelp.sql ... SUCCESS
  - Applying Updates/0245_blackrock_bracer.sql ... SUCCESS
  - Applying Updates/0246_WDB-5132_i.10305,10306,10307,10308,10309,10310_loot_tables.sql ... SUCCESS
  - Applying Updates/0247_WDB-5133_q.8305.sql ... SUCCESS
  - Applying Updates/0248_WDB-5134_c.13018.sql ... SUCCESS
  - Applying Updates/0249_WDB-5136_q.1667.sql ... SUCCESS
  - Applying Updates/0250_WDB-5141_c.507.sql ... SUCCESS
  - Applying Updates/0251_WDB-5145_c.15308.sql ... SUCCESS
  - Applying Updates/0252_WDB-4746_Barrens_KolkarPopulation.sql ... SUCCESS
  - Applying Updates/0252_WDB-5146_c.3273.sql ... SUCCESS
  - Applying Updates/0253_WDB-4740_c.4166.sql ... SUCCESS
  - Applying Updates/0254_WDB-5147_c.5797.sql ... SUCCESS
  - Applying Updates/0255_WDB-5148_c.12676.sql ... SUCCESS
  - Applying Updates/0256_WDB-5149_c.12903.sql ... SUCCESS
  - Applying Updates/0257_VDB-20200408191947_world.sql ... SUCCESS
  - Applying Updates/0258_kaelthas.sql ... SUCCESS
  - Applying Updates/0259_c.785.sql ... SUCCESS
  - Applying Updates/0260_WDB-5150_c.14479.sql ... SUCCESS
  - Applying Updates/0260_legendary_weapons_ai.sql ... SUCCESS
  - Applying Updates/0261_WDB-5151_c.14471.sql ... SUCCESS
  - Applying Updates/0262_WDB-5154_c.2848.sql ... SUCCESS
  - Applying Updates/0263_WDB-5137_c.2520.sql ... SUCCESS
  - Applying Updates/0263_WDB-5156_q.930.sql ... SUCCESS
  - Applying Updates/0264_q.1526.sql ... SUCCESS
  - Applying Updates/0265_c.4961.sql ... SUCCESS
  - Applying Updates/0266_mangos_string_near.sql ... SUCCESS
  - Applying Updates/0267_c.10698.sql ... SUCCESS
  - Applying Updates/0268_c.4479.sql ... SUCCESS
  - Applying Updates/0269_WDB-5167_speed.sql ... SUCCESS
  - Applying Updates/0270_WDB-5172_c.17704.sql ... SUCCESS
  - Applying Updates/0271_VDB-20231017014137_world.sql ... SUCCESS
  - Applying Updates/0272_SSC.sql ... SUCCESS
  - Applying Updates/0273_VDB-20231020080914_world.sql ... SUCCESS
  - Applying Updates/0274_kraator.sql ... SUCCESS
  - Applying Updates/0274_thrash_s.3391.sql ... SUCCESS
  - Applying Updates/0275_tbc_starter_zone_vein_overhaul.sql ... FAILED!
>>> ERROR 1064 (42000) at line 1041: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'a WHERE guid NOT IN(SELECT guid FROM gameobject b WHERE a.guid=b.guid) AND guid ' at line 1
Press space to continue...[H[2J[3JChecking minimum requirement please wait...
[H[2J[3JWelcome to the CMaNGOS TBC databases manager
============================================

Source version            : [1.10.0] Return of the Vengeance
Last source content update: [0291_npc_vendor_limited_stock]
Database content version  : []

Info: Your DBs seem to be up to date with core revision.

> 1) Manage settings
> 2) Install fresh TBC-DB only to 'tbcmangos'
> 3) Install core updates only
> 4) Full installation (create all DB and MySQL user, root required)
> 5) Advanced DB management (root required)
> 6) Manage realm list
> 7) Manage your DB backup
> 9) Quit

[?2004hPlease enter your choice.....: 9
[?2004l
```

### Core SHA1 Commit Hash

6c9da3dfa1fc72c84d3ab82d119e0990c48ed751

### Database SHA1 Commit Hash

dcf9967fe151a62623d78e961c006934b418998a

### Server Operating System

Windows 8.1 with Update 3 Professional 64bit

### Client Version

2.4.3 (The Burning Crusade)

### Client Operating System

Windows 8.1 with Update 3 Professional 64bit

**Comment by evil-at-wow** [comment]
> MySQL v5.7.41

And there's the reason for the error you get, becaue MySQL 5.7 does not support a 'table alias' in the delete statement.

In other words, you can write something like
> DELETE FROM pool_gameobject WHERE ...

but not

> DELETE FROM pool_gameobject __a__ WHERE ...

Of course, in this case there's a subquery that references the a (a.guid) so you can't just delete the a and be done. 

But let's look at that query for a second:
> DELETE FROM pool_gameobject a WHERE guid NOT IN(SELECT guid FROM gameobject b WHERE a.guid=b.guid) AND guid BETWEEN 75018 AND 75118;

This is obviously deleting rows from pool_gameobject where the condition following WHERE evaluates to true. To understand what it's doing exactly, let's take a random row in pool_gameobject with some guid X. The condition is then true when X is not in ```(SELECT guid FROM gameobject b WHERE X=b.guid)``` and between those two fixed values. But wait a second, that subquery is basically only going to be selecting one row at most, the one with guid being X (if there is such a row). So basically that query wants to delete a row from pool_gameobject when guid is between two given values and there is no row in gameobject with that guid. 

Well, there's an easier way to write that, and the nice benefit is this should work fine with MySQL 5.7 as well:
> DELETE FROM pool_gameobject WHERE guid NOT IN (SELECT guid FROM gameobject) AND guid BETWEEN 75018 AND 75118;

Sure, it now looks like the subquery is going to create a potentially large set of all guids that are used in gameobject, but that's not actually going to be the case for any decent database: they have long learned to optimize queries like this because the 'guid NOT IN (...)' part is what's called a dependent subquery. So this should have zero effect practically speaking.

In summary: try replacing lines 1041-1043 in that file by these queries that should do the same:
> DELETE FROM pool_gameobject WHERE guid NOT IN (SELECT guid FROM gameobject) AND guid BETWEEN 75018 AND 75118;
DELETE FROM pool_gameobject WHERE guid NOT IN (SELECT guid FROM gameobject) AND guid BETWEEN 5301069 AND 5301151;
DELETE FROM pool_gameobject WHERE guid NOT IN (SELECT guid FROM gameobject) AND guid BETWEEN 5301507 AND 5301531;

Please report back if that fixes your issue; we can update the SQL files then.

Offtopic: thinking about what those queries do, they seem a bit strange to me. They remove rows from pool_gameobject with guids that have no corresponding rows in gameobject?!? Now I always thought that the guid in pool_gameobject was a reference to rows in gameobject... so just maybe this query was wrong to start with?
@killerwife Do you remember what your intention was exactly with these queries?

**Comment by LEO33SoFo** [comment]
> Please report back if that fixes your issue; we can update the SQL files then.

@evil-at-wow I confirm your final query fixes the issue, thanks. :relaxed:

**Comment by killerwife** [comment]
The intention is simple. I wanted to delete pool entires for objects that have been deleted in my vein rework of tbc starter zones and i did not want to type 150 lines by hand :D

**Comment by evil-at-wow** [comment]
@LEO33SoFo A different fix was done than the one I originally proposed here - see cmangos/tbc-db@50f6aa1fd5251207d53a58a703d7169e7475c655 - but I think that should also fix it if one uses MySQL 5.7. So I'm going to close this issue, but feel free to ping here if that's not the case.
